app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.ts

```diff
@@ -341,10 +339,39 @@ export function addToTriggerGroup(
     },
   ];
 
+  const triggersGroupRunArgs = { ...triggersGroupRunArgsToAdd };
+
+  if (!!mapping && !!triggersGroupRunArgs) {
+    mapping.forEach((map) => {
+      const keySplit = map.key.split(':');
+      const currpPipelineId = {
```

Comment:
currp?

Reply:
ooops, typo.... will rename to `currentPipelineId `